src/Hint/Monad.hs

```diff
@@ -45,7 +51,9 @@ main = f $ do g a $ sleep 10 --
 main = do f a $ sleep 10 -- @Ignore
 main = do foo x; return 3; bar z --
 main = void $ forM_ f xs -- forM_ f xs
-main = void $ forM f xs -- void $ forM_ f xs
+main = void (forM_ f xs) -- (forM_ f xs)
```

Comment:
These brackets seem a bit odd?

Reply:
Probably, but this is orthogonal and unrelated to this change. `void (forM_ f xs)` -> `(forM_ f xs)` is the same as the previous behavior. The new behavior is `void (forM f xs)` -> `(forM_ f xs)`.

Reply:
@ndmitchell I hoped this addresses your question - let me know if you have any other concerns!

Reply:
Ah, didn't realise that was what it did before. Have fixed that separately in accc63710b17bf9b533288b3c0ba06ecd5c3be06